src/nodejs/hooks/sealights.go

```diff
@@ -149,13 +198,19 @@ func (sl *SealightsHook) usePackageJson(originalStartCommand string, stager *lib
 	return fmt.Errorf(NpmCommandStringError), false
 }
 
-func (sl *SealightsHook) getSealightsOptions(app string, token string, tokenFile string) *SealightsOptions {
-	o := &SealightsOptions{
-		Token:       token,
-		TokenFile:   tokenFile,
+func (sl *SealightsHook) getSealightsOptions(app string) *SealightsRunOptions {
+
+	proxy := os.Getenv("SL_PROXY")
```

Comment:
@CubikNeRubik like you said, please add the following here:

		BsId:        os.Getenv("SL_BUILD_SESSION_ID"),
		BsIdFile:    os.Getenv("SL_BUILD_SESSION_ID_FILE")
		LabId:       os.Getenv("SL_LAB_ID"),
		ProjectRoot: os.Getenv("SL_PROJECT_ROOT"),
		TestStage:   os.Getenv("SL_TEST_STAGE"),

In the same format like `SL_PROXY` and `sl.parameters.Proxy`